nym-api/nym-api-requests/src/ecash/models.rs

```diff
@@ -424,8 +600,9 @@ impl IssuedTicketbooksForResponseBody {
 pub struct IssuedTicketbooksForResponse {
     pub body: IssuedTicketbooksForResponseBody,
 
-    /// Signature on the body    
+    /// Signature on the body
     #[schemars(with = "PlaceholderJsonSchemaImpl")]
```

Comment:
I guess now we might be able to remove all those `    #[schemars(with = "PlaceholderJsonSchemaImpl")]` I put everywhere? : )

Reply:
done :heavy_check_mark: